examples/blinky/CMakeLists.txt

```diff
@@ -55,6 +55,7 @@ if(BLINKY_F4_EXAMPLE)
     target_link_libraries(stm32-blinky-f4
         HAL::STM32::F4::RCC
         HAL::STM32::F4::GPIO
+        HAL::STM32::F4::FLASH
```

Comment:
I believe there is a slight difference as now flash.c is compiled.
Before your change, only flash.h was included but flash.c not built.

Reply:
It's not an issue from my point of view we can live with it. Will see with @Hish15 

However, have a look at workflow failure